graphrag/index/flows/create_final_community_reports.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) 2024 Microsoft Corporation.
+# Licensed under the MIT License
+
+"""All the steps to transform community reports."""
+
+from uuid import uuid4
+
+import pandas as pd
+from datashaper import (
+    AsyncType,
+    VerbCallbacks,
+)
+
+from graphrag.index.cache import PipelineCache
+from graphrag.index.graph.extractors.community_reports.schemas import (
```

Comment:
I think I did a note to self in a previous PR, but, will add it here as well: Simplify this import